<!DOCTYPE html>
<html>
	<head>
		    <meta charset="utf-8">
		<title>CVE-2019-16226 - Vulnerability Data</title>
		    <link rel="shortcut icon" type="image/png" href="/favicon.ico" />
	</head>
<body>
<h2>Vulnerability Data</h2>
<h1>CVE-2019-16226</h1>
<p>An issue was discovered in py-lmdb 0.97. mdb_node_del does not validate a memmove in the case of an unexpected node->mn_hi, leading to an invalid write operation. NOTE: this outcome occurs when accessing a data.mdb file supplied by an attacker.</p>

<dl>
	<dt>Published</dt>
	<dd>2019-09-11</dd>

	<dt>Modified</dt>
	<dd>2019-10-12</dd>
</dl>

<h3>References</h3>
<ul>
	<li><a href="https://github.com/TeamSeri0us/pocs/tree/master/lmdb/lmdb%20memory%20corruption%20vuln">MISC:https://github.com/TeamSeri0us/pocs/tree/master/lmdb/lmdb%20memory%20corruption%20vuln</a></li>
</ul>

<footer>
	<p><a href="https://reconmap.org">Track security vulnerabilities</a> with <strong>Reconmap</strong>.</p>
</footer>
